amr-wind/turbulence/LES/Kosovic.cpp

```diff
@@ -0,0 +1,195 @@
+#include <cmath>
+
+#include "amr-wind/turbulence/LES/Kosovic.H"
+#include "amr-wind/turbulence/TurbModelDefs.H"
+#include "amr-wind/fvm/nonLinearSum.H"
+#include "amr-wind/fvm/strainrate.H"
+#include "amr-wind/fvm/divergence.H"
+#include "AMReX_REAL.H"
+#include "AMReX_MultiFab.H"
+#include "AMReX_ParmParse.H"
+
+namespace amr_wind {
+namespace turbulence {
+
+template <typename Transport>
+// cppcheck-suppress uninitMemberVar
+Kosovic<Transport>::Kosovic(CFDSim& sim)
+    : TurbModelBase<Transport>(sim)
+    , m_vel(sim.repo().get_field("velocity"))
+    , m_rho(sim.repo().get_field("density"))
+    , m_Nij(sim.repo().declare_field("Nij", 9, 1, 1))
+    , m_divNij(sim.repo().declare_field("divNij", 3, 1, 1))
+{
+    amrex::ParmParse pp("Kosovic");
+    pp.query("Cb", m_Cb);
+    m_Cs = std::sqrt(8 * (1 + m_Cb) / (27 * 3.14 * 3.14));
+    m_C1 = std::sqrt(960) * m_Cb / (7 * (1 + m_Cb) * m_Sk);
+    m_C2 = m_C1;
+    amrex::Print() << "C1:" << m_C1 << std::endl;
+    pp.query("refMOL", m_refMOL);
+    if (std::abs(m_refMOL) > 1000)
```

Comment:
I am surprised clang-format didn't put curly braces here... I though we enforced that.

Reply:
gets enforced through lint-clang-tidy